backend/app/services/llm/mappers.py

```
@@ -228,6 +228,54 @@ def map_kaapi_to_sarvam_params(kaapi_params: dict) -> tuple[dict, list[str]]:
+
+    Args:
+        kaapi_params: Dictionary with standardized Kaapi parameters
+
+    Returns:
+        Tuple of:
+        - Dictionary of SarvamAI API parameters
+        - List of warnings for unsupported parameters
+
+    """
+    elevenlabs_params = {}
+    warnings = []
```

Ruff (0.15.4)

[error] Local variable `warnings` is assigned to but never used

Remove assignment to unused variable `warnings`

(F841)